wgpu-core/src/command/compute.rs

```diff
@@ -897,33 +932,36 @@ impl Global {
         pass: &mut ComputePass<A>,
         offset: u32,
         data: &[u8],
-    ) {
-        assert_eq!(
-            offset & (wgt::PUSH_CONSTANT_ALIGNMENT - 1),
-            0,
-            "Push constant offset must be aligned to 4 bytes."
-        );
-        assert_eq!(
-            data.len() as u32 & (wgt::PUSH_CONSTANT_ALIGNMENT - 1),
-            0,
-            "Push constant size must be aligned to 4 bytes."
-        );
-        let value_offset = pass.base.push_constant_data.len().try_into().expect(
-            "Ran out of push constant space. Don't set 4gb of push constants per ComputePass.",
-        ); // TODO: make this an error that can be handled
```

Comment:
nitpick: I've moved the conversion of these panicking cases into errors as their own separate commit (see ecd969969).

No action needed here.